src/accelerate/utils/dataclasses.py

```diff
@@ -173,19 +173,53 @@ class InitProcessGroupKwargs(KwargsHandler):
 class FP8RecipeKwargs(KwargsHandler):
     """
     Use this object in your [`Accelerator`] to customize the initialization of the recipe for FP8 mixed precision
-    training. Please refer to the documentation of this
+    training with `transformers-engine`. Please refer to the documentation of this
```

Comment:
This makes it sound that TE is the only option, but MS-AMP is the default.

Reply:
Adjusted